client/client.go

```diff
@@ -38,32 +38,47 @@ type DataExportClient interface {
 
 // Client represents a HTTP client for making requests to the FullStory API.
 type Client struct {
-	HTTPClient *http.Client
-	Config     *config.Config
+	HTTPClient    *http.Client
+	Configuration *config.Config
```

Comment:
Why the rename here?

Reply:
That was reverted when I accepted cam's changes in #92